tests/backup/backup_resiliency_test.go

```diff
@@ -1198,3 +1199,253 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
+			dash.VerifyFatal(err, nil, "Getting admin context")
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating source cluster %s and destination cluster %s", SourceClusterName, destinationClusterName))
+			srcClusterStatus, _ = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			dash.VerifyFatal(srcClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying source cluster %s status", SourceClusterName))
+			destClusterStatus, destClusterUid = Inst().Backup.RegisterBackupCluster(orgID, destinationClusterName, "")
+			dash.VerifyFatal(destClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying destination cluster %s status", destinationClusterName))
+		})
+		Step("Taking backup of applications", func() {
+			log.InfoD("Taking backup of applications")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Getting admin context")
+			for _, namespace := range appNamespaces {
+				sem <- struct{}{}
+				backupName := fmt.Sprintf("%s-%s", BackupNamePrefix, namespace)
+				backupNames = append(backupNames, backupName)
+				wg.Add(1)
+				go func(backupName string, namespace string) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					defer func() { <-sem }()
+					// Here we are using destination cluster as application cluster
+					_, err = CreateBackupWithoutCheck(backupName, destinationClusterName, bkpLocationName, backupLocationUID,
+						[]string{namespace}, labelSelectors, orgID, destClusterUid, "", "", "", "", ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Taking backup %s of application- %s", backupName, namespace))
+				}(backupName, namespace)
+			}
+			wg.Wait()
+			log.Infof("The list of backups taken are: %v", backupNames)
+		})
+		Step("Reboot one worker node on application cluster when backup is in progress", func() {
+			log.InfoD("Switching cluster context to application[destination] cluster")
+			SetDestinationKubeConfig()
+			listOfWorkerNodes = node.GetWorkerNodes()
+			err := Inst().N.RebootNode(listOfWorkerNodes[0], node.RebootNodeOpts{
+				Force: true,
+				ConnectionOpts: node.ConnectionOpts{
+					Timeout:         1 * time.Minute,
+					TimeBeforeRetry: 5 * time.Second,
+				},
+			})
+			dash.VerifySafely(err, nil, fmt.Sprintf("Rebooting worker node %v", listOfWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context back to source cluster")
+			err = SetSourceKubeConfig()
+			dash.VerifyFatal(err, nil, "Switching back context to source cluster")
+
+		})
+		Step("Check if backup is successful after one worker node on application cluster is rebooted", func() {
+			log.InfoD("Check if backup is successful after one worker node on application cluster is rebooted")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, backupName := range backupNames {
+				backupStatus, err := backupSuccessCheck(backupName, orgID, 0, 0, ctx)
```

Comment:
Kindly keep an eye on pull request #1215. It will not be possible anymore to pass 0, 0 to backupSuccessCheck.

Reply:
@kphalgun-px 
Sure Will change it. 
Wanted to know the reason for removing the default values.

Reply:
@mkoppal-px suggested that setting them to 0, 0 would not provide a correct picture.